lib/ansible/modules/cloud/amazon/cloudwatchevent_rule.py

```diff
@@ -65,7 +65,9 @@
     description:
       - "A dictionary array of targets to add to or update for the rule, in the
         form C({ id: [string], arn: [string], role_arn: [string], input: [valid JSON string],
-        input_path: [valid JSONPath string], ecs_parameters: {task_definition_arn: [string], task_count: [int]}}).
+        input_path: [valid JSONPath string], ecs_parameters: {task_definition_arn: [string], task_count: [int]},
+        launch_type: 'FARGATE', network_configuration: {awsvpc_configuration: {subnets: [string],
+        security_groups: [string], assign_public_ip: 'ENABLED' }).
```

Comment:
assign_public_ip should also allow `DISABLED`.

```diff
-        security_groups: [string], assign_public_ip: 'ENABLED' }).
+        security_groups: [string], assign_public_ip: 'ENABLED|DISABLED' }).
```

Reply:
Thanks @opapy . That is now updated.